REPORT.md

```diff
@@ -0,0 +1,76 @@
+# Reporting capabilities within zAppBuild
+
+zAppBuild provides a set of reporting capabilities, which are part of the build framework itself to address some common demands of mainframe development teams.
+
+Developers are particularly interested if
+* a change of an element impacts other application components which are managed in a different repository.
+* their changes on an isolated feature branch potentially cause a conflict with some concurrent development done by others within the same repository.
+
+While during the analysis phase of a new task, the developer leverages application dependency understanding tools such as IBM Application Discovery, it is beneficial to enable also the build framework to automatically generate reports to answer the above questions.
+
+By default these reporting capabilities are turned off, while they don't belong to the core functionalities of a build framewoek.
+
+## Report External Impacts 
+
+### Purpose
+
+The _Reporting External Impacts_ feature enables the build framework to generate reports about impacted files in other application contexts to document cross application impacts for files that were changed within your application scope.
+
+### Pre-requisites
+
+Technically, this feature analyzes the list of changed files and queries the collection of other applications within the DBB WebApp, so it requires that the other applications are also processed through the CI/CD pipeline with DBB. It requires a repository connection.
```

Comment:
Suggestion:  "Technically, this feature analyzes the list of changed files **for the current application** and **then** queries the **collections** of other applications within the DBB WebApp.  **Therefore, to fully analyze cross application impacts, the other applications must also be part of and processed through the CI/CD pipeline with DBB**".

Question: "It requires a repository connection." - Which repository?  WebApp or GIT?  If GIT, I would conclude with the following.  "Additionally, access to the other application repositories is required.